html/pfappserver/lib/pfappserver/PacketFence/Controller/Config/Fingerbank/DB.pm

```diff
@@ -30,12 +30,10 @@ sub update :Local :Args(0) :AdminRole('FINGERBANK_UPDATE') {
 
     $c->stash->{current_view} = 'JSON';
 
-    my ( $status, $status_msg ) = fingerbank::DB::update_upstream;
-
-    pf::fingerbank::sync_upstream_db();
+    my $apiclient = pf::api::jsonrpcclient->new;
```

Comment:
bleh... pf::api::client::getClient() would have been better IMO.
Otherwise, I may expect a client shopping for some turkey stuff at Loblaws.
Anyway, will use it :P